src/dmd/globals.d

```diff
@@ -561,15 +561,19 @@ struct Loc
     const(char)* filename; // either absolute or relative to cwd
     uint linnum;
     uint charnum;
+    version(ExtraLocation)
+        uint offset;
 
     static immutable Loc initial;       /// use for default initialization of const ref Loc's
 
 nothrow:
-    extern (D) this(const(char)* filename, uint linnum, uint charnum) pure
+    extern (D) this(const(char)* filename, uint linnum, uint charnum, uint offset = 0) pure
```

Comment:
I don't mind, but Walter doesn't like parameters with default values. Is there a point to have this parameter since it's not used.